resources/user-agents/browsers/qt/qtwebengine-5-2-on.json

```diff
@@ -33,6 +33,15 @@
           "platforms": [
             "Linux_64", "Linux"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) QtWebEngine/#MAJORVER#.#MINORVER#*Chrome/*Safari/*",
```

Comment:
QT Web Engine has supported mac since at least version 5.5: https://doc.qt.io/qt-5.11/supported-platforms-and-configurations.html